addons/binding/org.openhab.binding.energenie/src/main/java/org/openhab/binding/energenie/internal/api/JSONResponseHandler.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.energenie.internal.api;
+
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.openhab.binding.energenie.internal.api.constants.JSONResponseConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonParser;
+
+/**
+ * Contains some common methods which are used for processing the server's
+ * response
+ *
+ * @author Mihaela Memova - Initial contribution
+ *
+ */
+public class JSONResponseHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(JSONResponseHandler.class);
```

Comment:
IMHO if it has to be `static final` it should be uppercased according to the naming standards. @kaikreuzer do you agree on that?